src/command/run.go

```diff
@@ -59,15 +71,35 @@ func RunWith(opts Options, cmd string, args ...string) (*Result, error) {
 	if opts.Env != nil {
 		subProcess.Env = opts.Env
 	}
-	output, err := subProcess.CombinedOutput()
+	var output bytes.Buffer
+	subProcess.Stdout = &output
+	subProcess.Stderr = &output
+	result := Result{command: cmd, args: args}
+	input, err := subProcess.StdinPipe()
+	if err != nil {
+		return &result, err
+	}
+	err = subProcess.Start()
+	if err != nil {
+		return &result, fmt.Errorf("can't start subprocess '%s %s': %w", cmd, strings.Join(args, " "), err)
```

Comment:
Finally, Go 1.13 provides a built-in way to wrap errors. I will convert al error wrapping in the code base to this new style in a separate PR.